### DISCUSSION TITLE
Hack tool on raw githubusercontent

### DISCUSSION THREAD

**Discussion by ScamCoin-Core** [community/community#81646] [New to GitHub] [answered]
Staff can contact me for the full link. They use github to distribute malware and steal chrome stored passwords.
<Edited by Staff>

**Comment by Akash1134** [comment, marked as answer]
Hi @ScamCoin-Core , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by ScamCoin-Core** [reply]
IT CAN'T BE FLAGGED SINCE IT'S A DIRECT LINK!!!
I have already mentioned this, but apparently someone didn't read what I actually wrote.

**Reply by allllllllllx** [reply]
this company is the creator of the system or program or whatever its. i been for two days trying to submit the god dammit abuse report and it took me into a lobe to end up with nothing 
i need a costumer support service for the repository abuse that been trying but for some reason there is nobody we have to go into the baffle of your site while actually not an expert.
[zoom feak.pdf](https://github.com/user-attachments/files/16324849/zoom.feak.pdf)

**Comment by ScamCoin-Core** [comment]
I have exhausted all possible ways of reporting this because it has a direct link to Github, therefore it cannot be flagged. For reference, here is the link https://raw.githubusercontent.com/nnshg/ggbvfd/main/Latest_App_797_PassKey.zip
________________________________

**Comment by attritionorg** [comment]
https://raw.githubusercontent.com/4ThePool/xmrig_setup/main/setup_4thepool_miner.sh
As well, and the GitHub support pages for reporting abuse give no easy way to report content on that domain.

**Comment by allllllllllx** [comment]
its irresponsible company every one should submit a complaint for FTC Federal Tread Commissioner, with cc to the IG office of your state and the Comptroller of the state. trust will answer i am giving them 24 hrs to solve it. if not i will make there legal department scream
